### PR TITLE
Update wookmark to v2.1.0 with npm auto-update config

### PR DESCRIPTION
**Pull request by dwong**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/germanysbestkeptsecret/Wookmark-jQuery
 * Official website (optional, not the repository): http://www.wookmark.com/jquery-plugin
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 2587
   - Count of watchers: 176
   - Count of forks: 849
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@dwong according to our document, would you please use auto-update config instead of manually update the lib? Thanks.

**Comment by dwong** [comment]
@PeterDaveHello does that work?

File: ajax/libs/jquery.wookmark/package.json [hunk 1]
```diff
@@ -12,5 +12,14 @@
...
   },
   "license": "MIT",
-  "author": "Christoph Ono <chri@sto.ph>"
+  "author": "Christoph Ono <chri@sto.ph>",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/germanysbestkeptsecret/Wookmark-jQuery.git",
+    "basePath": "",
+    "files": [
+      "wookmark.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.wookmark/package.json]: Would you please use `"wookmark*.js",` here? We'll like to add src files for popular libraries. Thanks.

**Comment by PeterDaveHello** [comment]
@dwong Thanks for update, I'll need to help me do few things:

1. Change the commit message in the first commit from `Update wookmark v2.1.0` to `Update wookmark to v2.1.0` or `Add wookmark v2.1.0`.
2. Update the auto-update config as I mentioned in https://github.com/cdnjs/cdnjs/pull/7570/files#r59834077 
3. Rebase your branch with our latest master branch.

Then I'll merge this, thanks for your contribution.

File: ajax/libs/jquery.wookmark/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/jquery.wookmark/package.json]: This lib also has npm package, would you please use npm auto-update config? It's more efficient.

**Comment by dwong** [comment]
@PeterDaveHello good to go?

**Comment by PeterDaveHello** [comment]
Nope, you changed the filemap to another things, we can not use that kind of wildcard in a non-dist folder, please use `wookmark*.js` and `css/main.css`

**Comment by PeterDaveHello** [comment]
cc #7454

**Comment by dwong** [comment]
@PeterDaveHello okay trying again!

**Comment by PeterDaveHello** [comment]
@dwong since we try to use `wookmark*.js`, would you please also add `wookmark.js` in this PR, we are almost there!

**Comment by dwong** [comment]
@PeterDaveHello okay I added `wookmark.js`.  Is it okay that I put `css/main.css` or should I have done `css/main.min.css`?

**Comment by PeterDaveHello** [comment]
@dwong both css/main.css and css/main.min.css, thanks.

**Comment by dwong** [comment]
@PeterDaveHello should I add `css/main.min.css` to the package.json as well?

**Comment by PeterDaveHello** [comment]
@dwong Nope, since there is no that file from upstream.

**Comment by dwong** [comment]
@PeterDaveHello okay added `css/main.css` as well

**Comment by PeterDaveHello** [comment]
Thanks @dwong

**Comment by dwong** [comment]
Thanks for your patience and help, @PeterDaveHello

**Comment by PeterDaveHello** [comment]
@dwong np

**Pull request merged**